public/scss/components/CartDetails.module.scss

```
@@ -131,41 +130,175 @@
   }
 }
 
+.itemPrice
+{
+  @include grid-start-(3,2);
+  align-self: center;
+}
+
 .itemSalePrice
 {
   @include SmallSemiBold;
-  color: $color_error
+  color: $color_error;
+}
+
+.itemQty
+{
+  @include grid-start-(2, 2);
 }
 
 .qtyBox
 {
-  @include SmallSmall;
-  display: flex;
-  align-items: center;
+  position: relative;
 
-  input
+  /* Chrome, Safari, Edge, Opera */
+  input::-webkit-outer-spin-button,
+  input::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+
+  /* Firefox */
+  input[type=number] {
+    -moz-appearance: textfield;
+  }
+
+  input[type="number"]
   {
+    @extend .form_input;
+    border: 1px solid $color_placeholder;
+    margin-bottom: 0;
     width: 100%;
-    border: none;
-    outline: none;
+    height: 56px;
+  }
+}
 
-    -moz-appearance:textfield;
+.changeQtyButton
+{
+  @extend .btn_transparent;
+  border: none !important;
+  outline: none !important;
+  padding: 0;
+  width: 10px;
 
-    &::-webkit-outer-spin-button,
-    &::-webkit-inner-spin-button 
+  &:first-child
+  {
+    @include position(absolute, auto, 10px, 8px);
+    font-size: 0;
+    @include fixedSize(12px, true);
+    background-repeat: no-repeat;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='6' viewBox='0 0 10 6' fill='none'%3E%3Cpath d='M4.99984 5.99977L0.756836 1.75677L2.17184 0.342773L4.99984 3.17177L7.82784 0.342773L9.24284 1.75677L4.99984 5.99977Z' fill='%23222222'/%3E%3C/svg%3E");
+  }
+  
+  &:last-child
+  {
+    @include position(absolute, 13px, 10px);
+    font-size: 0;
+    @include fixedSize(12px, true);
+    background-repeat: no-repeat;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='6' viewBox='0 0 10 6' fill='none'%3E%3Cpath d='M4.99984 2.828L2.17184 5.657L0.756836 4.243L4.99984 0L9.24284 4.243L7.82784 5.657L4.99984 2.828Z' fill='%23222222'/%3E%3C/svg%3E");
+  }
+
+  &:disabled
+  {
+    pointer-events: none;
+    cursor: not-allowed;
+    border-color: $color_white !important;
+    outline: none !important;
+  }
+}
+
+.itemNote
+{
+  grid-row: 5;
+  grid-column: 2/4;
+
+  @include flex(column, left, left);
+  @include typographyBuilder(600, 12, 16);
```

```suggestion
  @include flex(column, left, left);
  @include typographyBuilder(600, 12, 16);
  
  grid-row: 5;
  grid-column: 2/4;
```

per-include-an ditarok di paling atas ngab